scripts/gen-upgrade-proposal.sh

```diff
@@ -1,43 +1,45 @@
 #! /bin/bash
 set -ueo pipefail
 
-cat << EOF 1>&2
-------------------------------------------------------------
+cat <<'EOF' 1>&2
+------------------------------------------------------------------------
 This script shows a command for a software upgrade proposal which is
 compatible with the Cosmovisor found at:
 https://github.com/agoric-labs/cosmos-sdk/tree/Agoric/cosmovisor#readme
-------------------------------------------------------------
+------------------------------------------------------------------------
 EOF
 
 COMMIT_ID=$(git rev-parse HEAD)
-ZIPURL=https://github.com/Agoric/agoric-sdk/archive/$COMMIT_ID.zip
+ZIP_URL="https://github.com/Agoric/agoric-sdk/archive/${COMMIT_ID}.zip"
 
-echo "Verifying archive is at $ZIPURL..." 1>&2
+echo "Verifying archive is at $ZIP_URL..." 1>&2
 zipfile=$(mktemp)
 trap 'rm -f "$zipfile"' EXIT
-curl -L "$ZIPURL" -o "$zipfile"
+curl -L "$ZIP_URL" -o "$zipfile"
 
 echo "Generating SHA-256 checksum..." 1>&2
-checksum=sha256:$(shasum -a 256 "$zipfile" | cut -d' ' -f1)
+CHECKSUM=sha256:$(shasum -a 256 "$zipfile" | cut -d' ' -f1)
 
-info="{\"binaries\":{\"any\":\"$ZIPURL//agoric-sdk-$COMMIT_ID?checksum=$checksum\"},\"source\":\"$ZIPURL?checksum=$checksum\"}"
+UPGRADE_INFO="{\"binaries\":{\"any\":\"$ZIP_URL//agoric-sdk-${COMMIT_ID}?checksum=$CHECKSUM\"},\"source\":\"$ZIP_URL?checksum=$CHECKSUM\"}"
```

Comment:
This aligns with recent forum posts, but we could also quote all interpolated variables (both here and in future posts):
```suggestion
UPGRADE_INFO="{\"binaries\":{\"any\":\"${ZIP_URL}//agoric-sdk-${COMMIT_ID}?checksum=${CHECKSUM}\"},\"source\":\"${ZIP_URL}?checksum=${CHECKSUM}\"}"
```

Reply:
As you wish.  It may help readability, though is semantically identical, AFAIK.

Reply:
It is. Such a change would only be for consistency and readability.